src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy;
 
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
-import mate.academy.service.MovieService;
+import mate.academy.model.MovieSession;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        MovieSession movieSession = new MovieSession();
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
-        movieService.add(fastAndFurious);
-        System.out.println(movieService.get(fastAndFurious.getId()));
-        movieService.getAll().forEach(System.out::println);
+        movieSession.setMovie(fastAndFurious);
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setCapacity(150);
+        cinemaHall.setDescription("Great hall with soft chairs");
+        movieSession.setCinemaHall(cinemaHall);
+        LocalDateTime localDateTime = LocalDateTime.now();
+        movieSession.setShowTime(localDateTime);
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
+        movieSessionService.add(movieSession);
+
```

Comment:
Please use and test all services methods: add, get, getAll, findAvailableSessions